editor/editor_sectioned_inspector.cpp

```diff
@@ -245,16 +245,16 @@ void SectionedInspector::update_category_list() {
 		if (pi.name.find(":") != -1 || pi.name == "script" || pi.name == "resource_name" || pi.name == "resource_path" || pi.name == "resource_local_to_scene" || pi.name.begins_with("_global_script"))
 			continue;
 
+		if (!filter.empty() && pi.name.to_lower().find(filter.to_lower()) == -1)
```

Comment:
The problem with this approach is that it will not work if users search properties using spaces, i.e. with their `capitalize()` display instead of their internal snake_case path.

IMO it would be better to do this:
```
if (!filter.empty() && !filter.is_subsequence_ofi(pi.name) && !filter.is_subsequence_ofi(pi.name.capitalize()))
```
To support both use cases.

Reply:
Yeah, you're right, I didn't think about that.
I'm doubtful about using `is_subsequence_ofi` though, if `filter` is `"windows"` and `pi.name` is `"display/window/size/width"` it shows the `Window` category, I wouldn't expect this behaviour. 

Also with the current code it won't show any property for the `Window` category, it might be related to the change in `editor_inspector.cpp` though.

Reply:
> I'm doubtful about using `is_subsequence_ofi` though, if `filter` is `"windows"` and `pi.name` is `"display/window/size/width"` it shows the `Window` category, I wouldn't expect this behaviour.

On the other hand it means that you can search `"winwid"` and find `"display/window/size/width"` as the (likely) only match, which provides an easy way to filter properties when you have a rough idea of which one you're looking for.

I'm not strongly in favor of one or the other, but I figured that since previous code used `is_subsequence_ofi`, it might have been on purpose.

BTW instead of using `.capitalize()`, we could also do something like `.replace("/", " ").replace("_", " ")`, which would turn `"display/window/size/test_width"` into `display window size test width`. (Alternatively `.replace("/", " ").capitalize()` works too, gives `Display Window Size Test Width`.)

Reply:
Ok for me. :+1: